homework_3/homework_03.py

```diff
@@ -0,0 +1,126 @@
+    # зрозуміло дитині, що навчається в п'ятому класі
+"""
+
+# task 04
+print('''
+Площа Чорного моря становить 436 402 км2, а площа Азовського
+моря становить 37 800 км2. Яку площу займають Чорне та Азов-
+ське моря разом?''')
+s_black_sea = 436402
+s_azov_sea = 37800
+s_sea = s_black_sea + s_azov_sea
+
+print("Відповідь:", s_sea, "км2")
+
+# task 05
+print("""
+Мережа супермаркетів має 3 склади, де всього розміщено
+375 291 товар. На першому та другому складах перебуває
+250 449 товарів. На другому та третьому – 222 950 товарів.
+Знайдіть кількість товарів, що розміщені на кожному складі.
+""")
+goods_all = 375291
+storage_1_2 = 250449
+storage_2_3 = 222950
+storage_1 = goods_all - storage_2_3
+storage_2 = storage_1_2 - storage_1
+storage_3 = storage_2_3 - storage_2
+print("Відповідь: На першому складі було", storage_1, "товарів, на другому",
+      storage_2, "товарів, на третьому", storage_3, "товарів.")
+
+# task 06
+print("""
+Михайло разом з батьками вирішили купити комп’ютер, ско-
+риставшись послугою «Оплата частинами». Відомо, що сплачу-
+вати необхідно буде півтора року по 1179 грн/місяць. Обчисліть
+вартість комп’ютера.
+""")
+month = 18
+monthly_fee = 1179
+cost = month * monthly_fee
+print("Відповідь:", cost,"грн")
+
+# task 07
+print("""
+Знайди остачу від діленя чисел:
+a) 8019 : 8     d) 7248 : 6
+b) 9907 : 9     e) 7128 : 5
+c) 2789 : 5     f) 19224 : 9
+""")
+a = 8019 % 8
+b = 9907 % 9
+c = 2789 % 5
+d = 7248 % 6
+e = 7128 % 5
+f = 7128 % 5
+print("Відповідь: a)", a,", b)", b,
+      ", c)", c, ", d)", d,
+       ", e)", e, ", f)", f)
+
+# task 08
+print("""
+Іринка, готуючись до свого дня народження, склала список того,
+що їй потрібно замовити. Обчисліть, скільки грошей знадобиться
+для даного її замовлення.
+Назва товару    Кількість   Ціна
+Піца велика     4           274 грн
+Піца середня    2           218 грн
+Сік             4           35 грн
+Торт            1           350 грн
+Вода            3           21 грн
+""")
+pizza_big = 4*274
+pizza_average = 2*218
+juice = 4*35
+cake = 1*350
+water = 3*21
+cash = pizza_big + pizza_average + juice + cake + water
+print("Відповідь:", cash, "грн. знадобиться для даного її замовлення")
+# task 09
+print("""
+Ігор займається фотографією. Він вирішив зібрати всі свої 232
+фотографії та вклеїти в альбом. На одній сторінці може бути
+розміщено щонайбільше 8 фото. Скільки сторінок знадобиться
+Ігорю, щоб вклеїти всі фото?
+""")
+page = 232/8
+print("Відповідь:", page, "сторінок знадобиться Ігорю, щоб вклеїти всі фото.")
+
+# task 10
+print("""
+Родина зібралася в автомобільну подорож із Харкова в Буда-
+пешт. Відстань між цими містами становить 1600 км. Відомо,
+що на кожні 100 км необхідно 9 літрів бензину. Місткість баку
+становить 48 літрів.
+1) Скільки літрів бензину знадобиться для такої подорожі?
+2) Скільки щонайменше разів родині необхідно заїхати на зап-
+равку під час цієї подорожі, кожного разу заправляючи пов-
+ний бак?
+""")
+liters_gasoline = 1600/100*9
```

Comment:
It is better to cover by brackets more priorities actions to avoid unexpected prioritization of arithmetic calculations. 
For ex - (1600 / 100) * 9